### PR TITLE
fix(init): mongoDb fixes for prisma init

### PR DESCRIPTION
**Pull request by Jolg42**
Fixes to prisma init after trying it
- use mongoDb instead of lowercase version for preview flag
- missing model suggestion before I guess, I replaced the id in screenshot by the mongo equivalent we suggest in docs
 `id    String  @id @default(dbgenerated()) @map("_id") @db.ObjectId`

### CODE REVIEW

File: src/packages/sdk/src/getGenerators.ts [hunk 1]
```diff
@@ -147,11 +150,16 @@ export async function getGenerators({
   })
 
   if (dmmf.datamodel.models.length === 0) {
+    // Should have a condition if MongoDB datasource to have @id @map("_id")
```
Comment (Jolg42) [inline src/packages/sdk/src/getGenerators.ts:153]:
```suggestion
```

File: src/packages/sdk/src/getGenerators.ts [hunk 2]
```diff
@@ -147,11 +150,16 @@ export async function getGenerators({
...
+    if (config.datasources.some((d) => d.provider.includes('mongodb'))) {
+      throw new Error(missingModelMessageMongoDB)
+    }
+
     throw new Error(missingModelMessage)
   }
 
   if (
-    config.datasources.some((d) => d.provider.includes('mongoDb')) &&
+    config.datasources.some((d) => d.provider.includes('mongodb')) &&
```
Comment (millsp) [inline src/packages/sdk/src/getGenerators.ts:162]: Do we lowercase the providers?

**Review by millsp — APPROVED** [review]
Looks good! However, I'd like to see `missingGeneratorMessage.ts` split into multiple files (maybe another time).

File: src/packages/sdk/src/getGenerators.ts (hunk 2 above)
Comment (Jolg42) [reply src/packages/sdk/src/getGenerators.ts:162]: From what I tried uppercase even fails validation in the Prisma extension.

**Pull request merged**